github/repos_deployments_branch_policies_test.go

```diff
@@ -0,0 +1,153 @@
+package github

```

Comment:
Please add the standard copyright header (as seen in other files) followed by a blank line to this file and use the year 2023.